fastlane/Fastfile

```diff
@@ -34,7 +34,7 @@ ALL_BUNDLE_IDENTIFIERS = [
   'org.wordpress.WordPressAllTimeWidget',
   'org.wordpress.WordPressThisWeekWidget',
   'org.wordpress.WordPressIntents'
-].freeze
+]
```

Comment:
This is a fix related to the ASC deployment which didn't make it into `develop` with beta 1.